buildscripts/observability-test/Dockerfile

```
@@ -17,7 +17,7 @@
 # Stage 1: Build the interop test client and server
 #
 
-FROM openjdk:11.0.16-jdk-slim-bullseye AS build
+FROM eclipse-temurin:11-jdk AS build
```

Curious why not jre here a well.
```suggestion
FROM eclipse-temurin:11-jre AS build
```

---

In the first stage of the build, we are compiling the Java client binary. I tried just the `-jre` and got the error that there's no jdk...

---

Ah, right. Forgot we build PSM interop on Kokoro, then just copying the result. Makes sense then.